File: python/svix/models/list_response_message_attempt_out.py
# this file is @generated
import typing as t

from .common import BaseModel
from .message_attempt_out import MessageAttemptOut


class ListResponseMessageAttemptOut(BaseModel):
    data: t.List[MessageAttemptOut]

    done: bool

    iterator: t.Optional[str]

    prev_iterator: t.Optional[str] = None

File: csharp/Svix/Models/ListResponseMessageAttemptOut.cs
// this file is @generated
using Newtonsoft.Json;

namespace Svix.Models
{
    public class ListResponseMessageAttemptOut : BaseModel
    {
        [JsonProperty("data", Required = Required.Always)]
        public required List<MessageAttemptOut> Data { get; set; }

        [JsonProperty("done", Required = Required.Always)]
        public required bool Done { get; set; }

        [JsonProperty("iterator")]
        public string? Iterator { get; set; } = null;

        [JsonProperty("prevIterator")]
        public string? PrevIterator { get; set; } = null;
    }
}
